### ISSUE TITLE
Issue with rendering array with locals

### ISSUE THREAD

**Issue by kortirso** [thoughtbot/props_template#20]
Hi, I need to render array with some partial and additional local, I use something like

```
json.array! @orders, partial: ['shared/order_item', locals: {employee: current_employee}] do
end
```
and inside of partial

`json.iid order_item.iid`

But it returns list of the same iids from one of the orders (with any partial attributes - all elements in response will be the same from one of the elements)

`[{ 'iid' => 1 }, { 'iid' => 1 }, { 'iid' => 1 }]`

if remove locals - everything works fine and iids are different
Am I doing something wrong by using locals?

@jho406 can you help please?

I added test to check such thing - https://github.com/kortirso/props_template/pull/1, and it fails by rendering the same elements

**Comment by jho406** [comment]
Hi @kortirso, welcome!

Thanks for Issue and reproduction PR. You're not doing anything wrong 😄. Based on your PR, it looks like a bug. I'll take a look as soon as I can.

**Comment by jho406** [comment]
@kortirso Issue has been merged, and updated in `v0.35.0` run a bundle update to grab the changes. Thanks again for reporting this.
